cllc-public-app/Controllers/ApplicationsController.cs

```diff
@@ -841,7 +841,7 @@ public async Task<IActionResult> CreateApplication([FromBody] ViewModels.Applica
                         AdoxioSaturdayopen = defaultHours.AdoxioSaturdayopen
                     };
                     _dynamicsClient.Hoursofservices.Create(newHours);
-                        
+
                 }
             }
             catch (HttpOperationException httpOperationException)
```

Comment:
This line 847 is a good example of the type of catch referred to in my comment below